### ISSUE TITLE
Command.unshift producing strange command

### ISSUE THREAD

**Issue by Rubikan** [jonmagic/grim#25]
When using a non default ghostscript_path in the imagemagick_processor, this line gets executed:

```
command.unshift("PATH=#{File.dirname(@ghostscript_path)}:#{ENV['PATH']}") if @ghostscript_path && @ghostscript_path != DefaultGhostScriptPath
```

what was the actual purpose of it? The only thing it does for me now on windows is producing a strange command where he tries to invoke my whole path with the imagemagick arguments.

**Comment by jonmagic** [comment]
The last time I touched that line was here I think: https://github.com/jonmagic/grim/commit/3671c1d72d83f47baf328dc9a389c02558634325

Then you have these commits that change it:
https://github.com/jonmagic/grim/commit/7fa3eab27800fd5d400d2b35b6553a432634811c#diff-acd95e3665359bdb0f9e7518583032c2
https://github.com/jonmagic/grim/commit/d8a21bc381f37e3383a0737752abcce2a308a200#diff-acd95e3665359bdb0f9e7518583032c2

Do you think you could write a failing test that displays the issue you described?

**Comment by Rubikan** [comment]
I created a gist since I didn't want to commit and push a failing test ;)

https://gist.github.com/Rubikan/47e78c147d7a1d93ac8b

I tried it on Mac OSX (created a symlink to gs named gstest) and Windows, but it only failed on Windows.
The error I get looks like this:
Errno::ENOENT: No such file or directory - PATH=.:###########

Instead of the hashes it prints out my whole PATH (or big chunks of it, I didn't check it in the whole)

**Comment by bjgordon** [comment]
FYI I have the same problem on windows, I fixed it by commenting out that line. With ImageMagick convert and ghostscript on my path, I just had to set the processor with `Grim.processor =  Grim::ImageMagickProcessor.new({:ghostscript_path => "gswin64c.exe"})`
The better fix would be modifying the PATH var in an OS-agnostic way, but that's beyond my Ruby-fu. 
Thanks for your work, I'm finding this project useful.

**Comment by jonmagic** [comment]
@bjgordon thanks for the report and sorry for the late reply. Definitely sounds like this could be handled better. I don't run windows so I'm kind of hoping someone else will figure it out on windows and submit a PR and then I can test it against osx and linux :sparkles:

**Comment by jonmagic** [comment]
This should be resolved by #27 now.
